src/Parts/ApplicationCommandInteractionDataOptionStructure.php

```diff
@@ -12,7 +12,7 @@ class ApplicationCommandInteractionDataOptionStructure
     public ApplicationCommandOptionTypes $type;
     public string|int|float|bool|null $value;
     /**
-     * @var \Ragnarok\Fenrir\Parts\ApplicationCommandInteractionDataOptionStructure[]
+     * @var ApplicationCommandInteractionDataOptionStructure[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)